site/src/pages/ManagementSettingsPage/GroupsPage/GroupPage.tsx

```diff
@@ -50,7 +50,7 @@ import { isEveryoneGroup } from "utils/groups";
 import { pageTitle } from "utils/page";
 
 export const GroupPage: FC = () => {
-  const { organization, groupName } = useParams() as {
+  const { organization = "default", groupName } = useParams() as {
```

Comment:
I see the constant "default" used in multiple places. We should define it in a `const.ts` file for consistency. Makes sense?

Reply:
personally, I really dread adding imports like that which end up being more typing that the thing you could just say directly